paper-menu-button.html

```diff
@@ -365,12 +365,13 @@
          * The content element that is contained by the menu button, if any.
          */
         get contentElement() {
+          // Polymer 2.x returns slot.assignedNodes which can contain text nodes.
           var nodes = Polymer.dom(this.$.content).getDistributedNodes();
-          var i = 0;
-          while (nodes[i] && nodes[i].nodeType !== Node.ELEMENT_NODE) {
```

Comment:
Yes, it was pushing `i` forward until it found an Element. Basically changed it to a `for` loop that returns as soon as it finds an Element